travis-autobuild-extensions.sh

```diff
@@ -11,7 +11,7 @@ if [ "$TRAVIS_REPO_SLUG" == "$GH_REPO" ] && [ "$TRAVIS_PULL_REQUEST" == "false"
   git merge master -m "Merge master $TRAVIS_COMMIT"
```

Comment:
can we change this to `git checkout -f` so that it's no longer *possible* for us to run into merge conflicts?

Reply:
I've changed it to `git reset --hard master` because I'm not sure of the exact behavior of `git checkout -f`